doc/rtd/templates/module_property.tmpl

```diff
@@ -0,0 +1,5 @@
+{% macro print_prop(name, types, description, prefix) -%}
```

Comment:
This is a much nicer change. These templates aren't very pretty, but much better to manage here than trying to format via the python code directly.